.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
@dapplion I didn't realize specifying both here would force this to duplicate the runs on "normal" pull requests.

I wonder if there's anything in the actions config that allows us to avoid this?

Reply:
Hope there was! For future reference: https://github.community/t5/GitHub-Actions/How-to-trigger-an-action-on-push-or-pull-request-but-not-both/td-p/35805